apps/pie-microsite/src/assets/styles/components/_list.scss

```diff
@@ -0,0 +1,50 @@
+@use '../fozzie-config' as f;
+
+.c-list {
+    list-style: none;
+    padding-left: 0;
+}
+
+.c-list--ordered {
+    counter-reset: item;
+
+    .c-list-item {
```

Comment:
<img width="307" alt="Screenshot 2023-01-05 at 15 52 39" src="https://user-images.githubusercontent.com/16766185/210823309-cab2bce1-7eda-47d5-a5ba-4eaac17b889f.png">
the pill items to not appear to wrap when their container shrinks